home-assignments/session1/ex1_ages_groups.py

```diff
@@ -0,0 +1,46 @@
+import json
+import yaml
+
+
+with open(r'C:\Users\HEZI LEVI\PycharmProjects\ops_assingment1\ppl_ages') as f:
+    data = json.load(f)
+
+
+
+def ppl_buckets(data):
+    ppl_ages = data['ppl_ages']
+
+    buckets = sorted(data['buckets'])
+
+    a = []
+    b = []
+    c = []
+    d = []
+
+    for dict_key, dict_value in ppl_ages.items():
```

Comment:
Meaningful variable names make it clear what your program is doing 
For example

```suggestion
    for name, age in ppl_ages.items():
```